courses/templates/course_detail.html

```diff
@@ -3,34 +3,44 @@
 
 {% block title %}MIT xPro | {{ course.title }}{% endblock %}
 
+{% block extrahead %}
+{{ block.super }}
+<style>
+  .header-block {
+    background-image: url('{{ course.background_image_mobile_url }}');
+  }
+
+  @media screen and (min-width: 1024px) { 
+    .header-block {
+      background-image: url('{{ course.background_image_url }}'); 
+    }
+  }
+</style>
+{% endblock %}
+
 {% block content %}
-<div class="course-detail">
-  <h1>{{ course.title }}</h1>
-  <h3>{{ course.description }}</h3>
-  <h3>Course Runs:</h3>
-  {% for course_run in course.courserun_set.all %}
-    <div style="background-color: {% cycle "#eee" "#fff" %}; padding: 10px;">
-      <h4 style="margin: 0 0 10px 0; padding: 0;">
-        {% if course_run.courseware_url %}
-            &#11016;
-            <a
-              href="{{ course_run.courseware_url }}"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              {{ course_run.title }}
-            </a>
-        {% else %}
-          {{ course_run.title }}
+<div class="header-block">
+  <div class="container header-container">
```

Comment:
Are both of these classes necessary? Is `container` meaningful in bootstrap?

Reply:
Yes, `container` is a bootstrap class used as a wrapper on content. The bootstrap grid system works inside a `container` or `container-fluid` div.